examples/dynamic-enums-example.yaml

```diff
@@ -0,0 +1,138 @@
+id: https://w3id.org/linkml/examples/enums
+title: Dynamic Enums Example
+name: dynamicenums-example
+description: This demonstrates the use of dynamic enums
+license: https://creativecommons.org/publicdomain/zero/1.0/
+
+prefixes:
+  linkml: https://w3id.org/linkml/
+  ex: https://w3id.org/linkml/examples/enums/
+  sh: https://w3id.org/shacl/
+  bioregistry: https://bioregistry.io/registry/
+  MONDO: http://purl.obolibrary.org/obo/MONDO_
+  NCIT: http://purl.obolibrary.org/obo/NCIT_
+  loinc: http://loinc.org/
+
+default_prefix: ex
+default_range: string
+
+default_curi_maps:
+    - semweb_context
+
+emit_prefixes:
+  - linkml
+  - rdf
+  - rdfs
+  - xsd
+  - owl
+
+imports:
+  - linkml:types
+
+  
+#==================================
+# Classes                         #
+#==================================
+
+classes:
+  HumanSample:
+    slots:
+      - name
+      - disease
+
+
+#==================================
+# Slots                           #
+#==================================
+
+slots:
+  name:
+    range: string
+  disease:
+    range: HumanDisease
+  vital_status:
+    enum_range:
+      permissible_values:
+        LIVING:
+        DEAD:
+        UNDEAD:
+
+#==================================
+# Enums
+#==================================
+
+enums:
+  Disease:
+    reachable_from:
+      source_ontology: bioregistry:mondo
+      source_nodes:
+        - MONDO:0000001 ## disease or disorder
+      is_direct: false
+      relationship_types:
+        - rdfs:subClassOf
+    minus:
+      permissible_values:
+        root_node:
+          meaning: MONDO:0000001 ## disease or disorder
+  HumanDisease:
+    description: Extends the Disease value set, including NCIT neoplasms, excluding non-human diseases
+    inherits:
+      - Disease
+    include:
+      - reachable_from:
+          source_ontology: bioregistry:ncit
+          source_nodes:
+            - NCIT:C3262
+    minus:
```

Comment:
Shouldn't minus be directly under include, or does minus have a meaning beyond removing terms included by include?

Reply:
it's a recursive structure, any enum structure can have any combo of include, minus, reachability queries

Semantics: IDs = (include + queries) - minus

Will be documented, meant to mark this as draft!

Reply:
Would `exclude` (the antonym of `include`) be better here than `minus`? Is there a reason why 'minus' was used here (I can only think that minus implied direct removal from the `include` constrained terms versus an `exclude` applied directly from the original include `source_(ontology/nodes)`?)